test/core/end2end/dualstack_socket_test.cc

```diff
@@ -169,7 +169,7 @@ void test_connect(const char* server_host, const char* client_host, int port,
   } else {
     /* Give up faster when failure is expected.
        BUG: Setting this to 1000 reveals a memory leak (b/18608927). */
-    deadline = ms_from_now(1500);
+    deadline = ms_from_now(3000);
```

Comment:
wouldn't we want to use https://github.com/grpc/grpc/blob/3538efb53bd6d3bfca91aa50db7a6e1b97b9c855/test/core/util/test_config.h#L34 ?